src/common/utils/url.js

```diff
@@ -40,7 +40,7 @@ function parseURL(inputURL) {
         return inputURL;
     }
     try {
-        return new URL(inputURL);
```

Comment:
can we add a unit test for the regex? so we can see what's a valid one and what are we filtering? that way if we find some other weird urls that cause us trouble we can make sure we don't reopen old problems